cloudfront-redirect/function.js

```diff
@@ -3,39 +3,57 @@
 const fs = require('fs').promises;
 
 let redirectMap = null;
+const FILE_NAME = '/tmp/redirect-sample.json';
 
 const aws = require('aws-sdk');
 const s3 = new aws.S3({ region: 'us-east-1' });
 const s3Params = {
   Bucket: 'redirectbuket',
```

Comment:
Could this typo interfere at some point, or are both s3Params just placeholders and will be replaced by real values later? (Github wouldn't let me comment on the actual commit, so I leave my comment here...)

Reply:
I believe they are just placeholders.